db/re/item_combo_db.txt

```diff
@@ -202,6 +202,7 @@
 2777:2778:5479,{ bonus bMaxHP,300; bonus bMatkRate,5; bonus2 bSubEle,Ele_Neutral,5; }
 2779:2780:5482,{ bonus bMatkRate,10; bonus bInt,5; bonus2 bSubRace,RC_Angel,10; }
 2779:2780:5577,{ bonus bMatkRate,10; bonus bInt,5; bonus2 bSubRace,RC_Angel,10; }
+2950:15067:20709:22012,{ bonus2 bExpAddClass,Class_All,10; }
```

Comment:
where do the `Class_All` come from ?
I can't find them in `doc\item_bonus.txt` or `db\const.txt`